kafka_consumer/datadog_checks/kafka_consumer/new_kafka_consumer.py

```diff
@@ -181,13 +182,23 @@ def _get_highwater_offsets(self):
                     ],
                 )
 
-                highwater_future = self.kafka_client._send_request_to_node(node_id=broker.nodeId, request=request)
+                # We can disable wakeup here because it is the same thread doing both polling and sending. Also, it
+                # is possible that the wakeup itself could block if a large number of sends were processed beforehand.
+                highwater_future = self._send_request_to_node(node_id=broker.nodeId, request=request, wakeup=False)
```

Comment:
Could you also add a comment to switch back to the official `_send_request_to_node` function once your other PR is merged and released?

Reply:
Just added!